docs/content/tutorials/virtual-server-with-custom-listener-ports.md

```diff
@@ -0,0 +1,171 @@
+---
+title: VirtualServer with Custom listener ports
+description: |
+  How to configure a VirtualServer with custom HTTP and HTTPS listener ports.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+---
+## Configuring a VirtualServer with custom HTTP and HTTPS listener ports.
+
+This tutorial outlines how to configure and deploy a VirtualServer resource to with custom HTTP and HTTPS listener ports.
+Previously this was task had to be performed manually as documented in [Customizing NGINX Ingress Controller Ports](/nginx-ingress-controller/tutorials/custom-listen-ports/).
+
+VirtualServer now explicitly defines custom HTTP and HTTPS listener ports using the `spec.listener.http` and `spec.listener.https` fields.
+Each field must reference a valid listener defined by in a [GlobalConfiguration](/nginx-ingress-controller/configuration/global-configuration/globalconfiguration-resource/) resource.
+
+## Deploy GlobalConfiguration
+
+1. Create a yaml file called `nginx-configuration.yaml` with the below content:
+```yaml
+apiVersion: k8s.nginx.org/v1alpha1
+kind: GlobalConfiguration
+metadata:
+  name: nginx-configuration
+  namespace: nginx-ingress
+spec:
+  listeners:
+  - name: http-8083
+    port: 8083
+    protocol: HTTP
+  - name: https-8443
+    port: 8443
+    protocol: HTTP
+    ssl: true
+```
+
+2. Deploy `nginx-configuration.yaml` file:
+```console
```

Comment:
@ADubhlaoich should we be using `shell` or `console` for snippets now?

Reply:
`shell`